src/cljs/main/broadfcui/common/flex_utils.cljs

```diff
@@ -4,12 +4,12 @@
     ))
 
 
-(defn flex-box [attrs & children]
+(defn box [attrs & children]
   [:div (utils/deep-merge {:style {:display "flex"}} attrs)
    children])
 
-(defn flex-strut [size]
+(defn strut [size]
   [:div {:style {:flexGrow 0 :flexShrink 0 :flexBasis size}}])
 
-(def flex-spacer
+(def spring
```

Comment:
Worth noting: If your parent says `{:justifyContent "space-between"}` you get a very similar effect to what this is used for. However, this might actually be cleaner.